test/test_individual.py

```diff
@@ -1,139 +1,165 @@
 import math
+import numpy as np
 import pickle
+from collections import namedtuple
+
 import pytest
 
 import cgp
-from cgp.individual import IndividualSingleGenome
+from cgp import IndividualSingleGenome, IndividualMultiGenome
 from cgp.genome import ID_INPUT_NODE, ID_OUTPUT_NODE, ID_NON_CODING_GENE
 
 
-def test_pickle_individual():
+TestParams = namedtuple("TestParams", ["genome_params", "primitives", "dna", "target_function"])
```

Comment:
why are you using a `namedtuple` instead of a dict here? is it much cleaner to offset the overhead of introducing it?

Reply:
also, having this nice modularity, shouldn't we use it in the tests, i.e., introduce an additional "TestParams" instance?

Reply:
We could use a dict, of course, but I think this is the perfect use case for a `NamedTuple`. By using a `NamedTuple`, the code for unpacking it in the tests, gets shorter, because we can treat it as a tuple.

Reply:
> also, having this nice modularity, shouldn't we use it in the tests, i.e., introduce an additional "TestParams" instance?

I don't understand what you mean here, sorry.

Reply:
well, you define a list of `TestParams` but right now this list only has one entry, e.g., tests only one graph version and target function. wouldn't it make sense to test multiple ones?

Reply:
Yes, but I would postpone that to a different PR. Let's not overload this one.

Reply:
Added #182 for this.